aip/client-libraries/4222.md

```diff
@@ -50,6 +50,19 @@ rpc CreateTopic(CreateTopicRequest) {
 }
 ```
 
+The value of the field `field` **must** be one of the following:
+1. a name of a field in the top-level of the request message
+1. a dot-separated path of field names leading to a field in a sub-message of
+the request message e.g. `"book.author.name"` where `book` is a message field in
+the request message, `author` is a message field in the `book` message, and
+`name` is a `string` field in the `author` message
+
+The _actual field_ specified in the field `field` **must** have the following
+characteristics:
+- it is type `string`
+- it has a path-like value format, resembling a resource name or a HTTP path
```

Comment:
I don't think this part is quite right. It *might* be a resource name, but e.g. BigTable's second routing parameter `app_profile_id` has no such structure and is just a (user-provided) string.

Reply:
For sure, I didn't nail that language. How about this?